testing/mod.ts

```diff
@@ -75,15 +76,13 @@ function createTestResults(tests: TestDefinition[]): TestResults {
   );
 }
 
-function report(result: TestResult): void {
+function report(result: TestResult, log: logginFunctionType): void {
   if (result.ok) {
-    console.log(`test ${result.name} ... ${GREEN_OK}`);
```

Comment:
A bit off-topic question, how come if test case uses `console.log` its output always goes between `test <name>` and `... OK`.

Eg:
```
test testPrettierCheckAndFormatFiles ./prettier/testdata/0.ts ... Not formatted
./prettier/testdata/1.js ... Not formatted
... ok
```

Seems like a bug, it'd be nice if test output was on new line as well.

Reply:
Prettier is not using console.log but direct stdout

Reply:
Other test cases suffer similar behavior:
```
test ioTempfile File { rid: 6 } /Users/biwanczuk/dev/deno_std/prefix-508272-postfix... ok
```

Reply:
the case of `ioTempFile` is weird yep.

Reply:
tried with rebase and i only have issues with prettier which uses `std.out`

```
Updated 0 paths from the index
.\prettier\testdata\0.ts ... Not formatted
.\prettier\testdata\1.js ... Not formatted
Updated 2 paths from the index
OK     testPrettierCheckAndFormatFiles
Updated 0 paths from the index
.\prettier\testdata\bar\0.ts ... Not formatted
.\prettier\testdata\bar\1.js ... Not formatted
.\prettier\testdata\foo\0.ts ... Not formatted
.\prettier\testdata\foo\1.js ... Not formatted
Updated 4 paths from the index
OK     testPrettierCheckAndFormatDirs
Updated 0 paths from the index
Updated 4 paths from the index
OK     testPrettierOptions
```